raft/src/test/java/org/apache/kafka/raft/RaftClientTestContext.java

```diff
@@ -641,19 +641,43 @@ void assertSentDescribeQuorumResponse(
         long highWatermark,
         List<ReplicaState> voterStates,
         List<ReplicaState> observerStates
+    ) {
+        short apiVersion = (short) (kip853Rpc ? 2 : 1);
+        assertSentDescribeQuorumResponse(leaderId, leaderEpoch, highWatermark, voterStates, observerStates, apiVersion, Errors.NONE);
+    }
+
+    void assertSentDescribeQuorumResponse(
+        int leaderId,
+        int leaderEpoch,
+        long highWatermark,
+        List<ReplicaState> voterStates,
+        List<ReplicaState> observerStates,
+        short apiVersion,
```

Comment:
I am sure you can remove this parameter if you look at my other comment about parametrizing `withKip853Rpc` and the call to the builder.

Reply:
I added this specifically for the test cases where I specify an older describeQuorumRequest version than what is default

Reply:
See my other comment but that is what `withKip853Rpc` is for. `false` uses the old version of the RPCs. `true` uses the new version of the RPCs.

Reply:
discussed offline, `withKip853Rpc` currently indicates more than what RPC versions the test should use (for instance if we should use non-zero directory ids), but I've convinced myself why that's fine. 
I'm removing `testDescribeQuorumOld` and will have the equiv functionality covered in `testDescribeQuorumWithFollowers(boolean withKip853Rpc=false)`